.stignore

```diff
@@ -0,0 +1,101 @@
+.git
```

Comment:
If I understand correctly this is a local configuration file of a tool you use on your development machine. Please remove it from the PR.

Reply:
This is a configuration file for Okteto. Okteto works by synchronizing local changes to a development container. This file indicates which files should not be synched.